docs/basics.txt

```diff
@@ -1169,7 +1198,8 @@ graph.tx().commit() // Can't fail due to uniqueName write lock contention involv
 
 Transactions are started automatically with the first operation executed against the graph. One does NOT have to start a transaction manually. The method `newTransaction` is used to start <<multi-thread-tx, multi-threaded transactions>> only.
 
-Transactions are automatically started under the TinkerPop semantics but *not* automatically terminated. Transactions have to be terminated manually with `g.commit()` if successful or `g.rollback()` if not. Manual termination of transactions is necessary because only the user knows the transactional boundary.
+Transactions are automatically started under the TinkerPop semantics but *not* automatically terminated. Transactions have to be terminated manually with `commit()` if successful or `rollback()` if not. Manual termination of transactions is necessary because only the user knows the transactional boundary.
```

Comment:
Just "terminated manually with `commit()` or `rollback()`"? Successful workflows might call "rollback()" and unsuccessful ones may call "commit()" (and "rollback()").